core/adminserver/admin_server.go

```diff
@@ -348,3 +348,37 @@ func (s *Server) DeleteDomain(ctx context.Context, in *pb.DeleteDomainRequest) (
 func (s *Server) UndeleteDomain(ctx context.Context, in *pb.UndeleteDomainRequest) (*google_protobuf.Empty, error) {
 	return nil, status.Errorf(codes.Unimplemented, "not implemented")
 }
+
+// GarbageCollect looks for domains that have been deleted for longer than a given duration and fully deletes them.
```

Comment:
s/for longer than a given duration/before the specified timestamp/

Reply:
done